mobilecoind-json/README.md

```diff
@@ -65,9 +65,45 @@ $ curl localhost:9090/monitors/a0cf8b79c9f8d74eb935ab4eeeb771f3809a408ad47246be4
 {"view_public_key":"543b376e9d5b949dd8694f065d95a98a89e6f17a20c621621a808605d1904324","spend_public_key":"58dba855a885dd535dc5180af443abae67c790b860d5adadb4d6a2ecb71abd28","fog_report_url":"","fog_authority_fingerprint_sig":"","fog_report_id":"", "b58_address_code": "7Q6gtA5EqSxkEsqsf5p2j7qEHkA8fBZYNsfuWTZTQaFAqo3FPo8PvhrrUobZfXagrLopzpxqxGBs7Hphwhsc56ryWriPWLCRadhRpnZW6AT"}
 ```
 
+### Simple payment flow
+There are two possible ways to make a payment. The simplest option is to use the intended recipient's `b58_address_code`, which they can get
+using the call above. 
+
+#### Initiate a transaction to a b58 public address
+This call initiates a transfer to a public address encoded as a b58 string. If the call returns successfully, the transaction has been
+submitted to the network. It returns receipts which you can use in calls below to determine if the transaction succeeded.
+
+```
+$ curl http://localhost:9090/monitors/%s/subaddresses/0/pay-address-code" -d '{"receiver_b58_address_code": "7Q6gtA5EqSxkEsqsf5p2j7qEHkA8fBZYNsfuWTZTQaFAqo3FPo8PvhrrUobZfXagrLopzpxqxGBs7Hphwhsc56ryWriPWLCRadhRpnZW6AT", "value": "1"}' -X POST -H 'Content-Type: application/json'
```

Comment:
Can we format these so that they don't render in a scroll box?

Reply:
Done, for the most part, some of the b58 codes are just too long and there's no way to break them up.